polaris-core/src/main/java/org/apache/polaris/core/storage/StorageAccessProperty.java

```diff
@@ -18,15 +18,23 @@
  */
 package org.apache.polaris.core.storage;
 
-/** Enum of polaris supported credential properties */
-public enum PolarisCredentialProperty {
+/**
+ * A subset of Iceberg catalog properties recognized by Polaris.
+ *
+ * <p>Most of these properties are meant to configure Iceberg FileIO objects for accessing data in
+ * storage.
+ */
+public enum StorageAccessProperty {
   AWS_KEY_ID(String.class, "s3.access-key-id", "the aws access key id"),
   AWS_SECRET_KEY(String.class, "s3.secret-access-key", "the aws access key secret"),
   AWS_TOKEN(String.class, "s3.session-token", "the aws scoped access token"),
   AWS_SESSION_TOKEN_EXPIRES_AT_MS(
```

Comment:
Is this considered "credential"?

Reply:
The Key ID, Secret Key, Session Token, and expiry time is definitely together considered a "credential". Not sure if there's a reason why you're asking this?

Reply:
I'd say it is part of the credential bunch of properties.